### PR TITLE
Uifc 138 resolve link to organization

### PR DESCRIPTION
**Pull request by elsenhans**

### CODE REVIEW

File: src/components/MetadataSources/MetadataSourceView.js [hunk 1]
```diff
@@ -16,14 +16,22 @@ import {
...
-  IfPermission
+  IfPermission,
 } from '@folio/stripes/core';
 
 import SourceInfoView from './SourceInfo/SourceInfoView';
 import SourceManagementView from './SourceManagement/SourceManagementView';
 import SourceTechnicalView from './SourceTechnical/SourceTechnicalView';
 
 class MetadataSourceView extends React.Component {
+  static manifest = Object.freeze({
```
Comment (rchr) [inline src/components/MetadataSources/MetadataSourceView.js:27]: Please remove commented-out code.
In addition, the manifest seems to be unused, thus you can delete it.

File: src/routes/SourceViewRoute.js [hunk 2]
```diff
@@ -14,6 +14,10 @@ class SourceViewRoute extends React.Component {
       type: 'okapi',
       path: 'finc-config/metadata-sources/:{id}',
     },
+    // org: {
```
Comment (rchr) [inline src/routes/SourceViewRoute.js:17]: Please remove commented-out code.

File: src/routes/SourceViewRoute.js [hunk 3]
```diff
@@ -28,6 +32,7 @@ class SourceViewRoute extends React.Component {
     }).isRequired,
     resources: PropTypes.shape({
       source: PropTypes.object,
+      // org: PropTypes.object,
```
Comment (rchr) [inline src/routes/SourceViewRoute.js:35]: Please remove commented-out code.

File: src/routes/SourceViewRoute.js [hunk 4]
```diff
@@ -63,6 +74,9 @@ class SourceViewRoute extends React.Component {
         }}
         isLoading={_.get(this.props.resources, 'source.isPending', true)}
         record={_.get(this.props.resources, 'source.records', []).find(i => i.id === this.props.match.params.id)}
+        stripes={this.props.stripes}
+        organizationId={_.get(this.props.resources, 'source.records.organization.id')}
+        // org={org}
```
Comment (rchr) [inline src/routes/SourceViewRoute.js:79]: Please remove commented-out code.

File: src/routes/SourceViewRoute.js [hunk 5]
```diff
@@ -54,6 +59,12 @@ class SourceViewRoute extends React.Component {
     const { handlers } = this.props;
     // const selectedRecord = this.getRecord(this.props.match.params.id);
 
+    // const organizationId = _.get(this.props.resources, 'source', '-');
```
Comment (rchr) [inline src/routes/SourceViewRoute.js:62]: Please remove commented-out code.

File: src/components/MetadataSources/MetadataSourceView.js [hunk 6]
```diff
@@ -32,6 +40,9 @@ class MetadataSourceView extends React.Component {
     isLoading: PropTypes.bool,
     record: PropTypes.object,
     stripes: PropTypes.object,
+    resources: PropTypes.shape({
```
Comment (rchr) [inline src/components/MetadataSources/MetadataSourceView.js:43]: If the manifest from above is not used, you may delete the unused propType `resources`.

File: src/components/MetadataSources/MetadataSourceView.js (hunk 1 above)
Comment (elsenhans) [reply src/components/MetadataSources/MetadataSourceView.js]: manifest has been removed

File: src/components/MetadataSources/MetadataSourceView.js (hunk 6 above)
Comment (elsenhans) [reply src/components/MetadataSources/MetadataSourceView.js]: unused prop resources has been removed

File: src/routes/SourceViewRoute.js (hunk 2 above)
Comment (elsenhans) [reply src/routes/SourceViewRoute.js]: commented-out code has been removed

File: src/routes/SourceViewRoute.js (hunk 4 above)
Comment (elsenhans) [reply src/routes/SourceViewRoute.js]: commented-out code has been removed

File: src/routes/SourceViewRoute.js (hunk 5 above)
Comment (elsenhans) [reply src/routes/SourceViewRoute.js]: commented-out code has been removed

File: src/routes/SourceViewRoute.js (hunk 3 above)
Comment (elsenhans) [reply src/routes/SourceViewRoute.js]: commented-out code has been removed

**Pull request merged**